pkg/tenant/projects_test.go

```diff
@@ -111,64 +111,6 @@ func (suite *tenantTestSuite) TestTenantProjectList() {
 	}
 }
 
-func (suite *tenantTestSuite) TestTenantProjectCreate() {
```

Comment:
Got moved in the merge.

Reply:
Ok - I trust that this is ok; thank you for making a note of it!